aten/src/ATen/core/ATen_fwd.h

```diff
@@ -4,8 +4,6 @@
 // Forward declarations of core ATen types used in dispatch functions
 namespace c10 {
 
-template<typename T>
-class optional;
```

Comment:
Should we just say something like `using optional = std::optional;` and kill `c10/util/Optional` header?

Reply:
that'd be fine with me, but I've had to leave in `#include <c10/macros/Macros.h>` in `c10/util/Optional.h` for now because lots of files seem to depend on pulling in the `Macros.h` namespace shenanigans through `Optional.h` :(

Reply:
Agre with @malfet . This is how llvm handled their std::optional conversion.